test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -214,6 +214,13 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 	},
 	// --
 
+	// k8s.io/kubernetes/pkg/apis/networking/v1
+	gvr("networking.k8s.io", "v1", "networkpolicies"): {
+		stub:             `{"metadata": {"name": "np1"}, "spec": {"podSelector": {"matchLabels": {"e": "f"}}}}`,
+		expectedEtcdPath: "/registry/networkpolicies/etcdstoragepathtestnamespace/np1",
+	},
```

Comment:
> Do we need to set them as CohabitatingResources?

Absolutely, yes. I expected this test to help catch cases where we weren't setting that properly